mongodbatlas/fw_data_source_mongodbatlas_alert_configuration.go

```diff
@@ -321,124 +322,111 @@ func outputAlertConfiguration(alert *matlas.AlertConfiguration, outputType, reso
+func outputAlertConfigurationResourceImport(label string, alert *admin.GroupAlertsConfig) string {
+	return fmt.Sprintf("terraform import mongodbatlas_alert_configuration.%s %s-%s\n", label, *alert.GroupId, *alert.Id)
 }
 
-func convertMatcherToCtyValues(matcher matlas.Matcher) map[string]cty.Value {
+func convertMatcherToCtyValues(matcher map[string]interface{}) map[string]cty.Value {
+	fieldName, _ := matcher["fieldName"].(string)
+	operator, _ := matcher["operator"].(string)
+	value, _ := matcher["value"].(string)
 	return map[string]cty.Value{
-		"field_name": cty.StringVal(matcher.FieldName),
-		"operator":   cty.StringVal(matcher.Operator),
-		"value":      cty.StringVal(matcher.Value),
+		"field_name": cty.StringVal(fieldName),
+		"operator":   cty.StringVal(operator),
+		"value":      cty.StringVal(value),
 	}
 }
 
-func convertMetricThresholdToCtyValues(metric matlas.MetricThreshold) map[string]cty.Value {
+func convertMetricThresholdToCtyValues(metric admin.ServerlessMetricThreshold) map[string]cty.Value {
 	return map[string]cty.Value{
-		"metric_name": cty.StringVal(metric.MetricName),
-		"operator":    cty.StringVal(metric.Operator),
-		"threshold":   cty.NumberFloatVal(metric.Threshold),
-		"units":       cty.StringVal(metric.Units),
-		"mode":        cty.StringVal(metric.Mode),
+		"metric_name": cty.StringVal(*metric.MetricName),
+		"operator":    cty.StringVal(*metric.Operator),
+		"threshold":   cty.NumberFloatVal(*metric.Threshold),
+		"units":       cty.StringVal(*metric.Units),
+		"mode":        cty.StringVal(*metric.Mode),
 	}
 }
 
-func convertThresholdToCtyValues(threshold matlas.Threshold) map[string]cty.Value {
+func convertThresholdToCtyValues(threshold *admin.GreaterThanRawThreshold) map[string]cty.Value {
 	return map[string]cty.Value{
-		"operator":  cty.StringVal(threshold.Operator),
-		"units":     cty.StringVal(threshold.Units),
-		"threshold": cty.NumberFloatVal(threshold.Threshold),
+		"operator":  cty.StringVal(*threshold.Operator),
+		"units":     cty.StringVal(*threshold.Units),
+		"threshold": cty.NumberFloatVal(float64(*threshold.Threshold)), // int in new SDK but keeping float64 for backward compatibility
 	}
 }
 
-func convertNotificationToCtyValues(notification *matlas.Notification) map[string]cty.Value {
+func convertNotificationToCtyValues(notification *admin.AlertsNotificationRootForGroup) map[string]cty.Value {
 	values := map[string]cty.Value{}
 
-	if notification.ChannelName != "" {
-		values["channel_name"] = cty.StringVal(notification.ChannelName)
-	}
-
-	if notification.DatadogRegion != "" {
-		values["datadog_region"] = cty.StringVal(notification.DatadogRegion)
-	}
-
-	if notification.EmailAddress != "" {
-		values["email_address"] = cty.StringVal(notification.EmailAddress)
+	if conversion.IsStringPresent(notification.ChannelName) {
+		values["channel_name"] = cty.StringVal(*notification.ChannelName)
 	}
 
-	if notification.FlowName != "" {
```

Comment:
has this field been removed? I don't see FlowName in the new code

Reply:
correct, they are for a notification type that doesn't exist any more